packages/backend/src/nest/libp2p/libp2p.service.ts

```diff
@@ -153,21 +162,22 @@ export class Libp2pService extends EventEmitter {
     this.processInChunksService.updateQueue(toDial)
   }
 
-  public async createInstance(params: Libp2pNodeParams): Promise<Libp2p> {
+  public async createInstance(params: Libp2pNodeParams, startDialImmediately: boolean = false): Promise<Libp2p> {
     if (this.libp2pInstance) {
       return this.libp2pInstance
     }
 
     let libp2p: Libp2p
+    const maxParallelDials = 2
 
     try {
       libp2p = await createLibp2p({
         start: false,
         connectionManager: {
-          minConnections: 3, // TODO: increase?
+          minConnections: 5, // TODO: increase?
           maxConnections: 20, // TODO: increase?
-          dialTimeout: 120_000,
-          maxParallelDials: 10,
+          dialTimeout: 120000,
+          maxParallelDials,
```

Comment:
What does this set maxParallelDials to? We observed a while back that too many dials or too many connections could create a performance problem, perhaps only on mobile. I think the performance problem was with Tor if I remember correctly. Maybe that's been fixed in Tor and it's not a problem anymore but we should check to make sure.

Reply:
This PR lowers the max parallel dials from 10 to 2 because it seems to cause issues with connections.